### PR TITLE
pages*: add missing mnemonics

### PR DESCRIPTION
**Pull request by vitorhcl**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages/android/am.md [hunk 1]
```diff
@@ -3,11 +3,11 @@
 > Android activity manager.
 > More information: <https://developer.android.com/studio/command-line/adb#am>.
 
-- Start a specific activity:
+- Start an activity with the specific component and package [n]ames:
```
Comment (acuteenvy) [inline pages/android/am.md:6]: There's only one name in this example.
```suggestion
- Start an activity with a specific component and package [n]ame:
```

File: pages/common/bedtools.md [hunk 2]
```diff
@@ -4,26 +4,26 @@
...
-`bedtools intersect -a {{path/to/file1}} -b {{path/to/file2}} -lof > {{path/to/output_file}}`
+`bedtools intersect -a {{path/to/file1}} -b {{path/to/file2}} -loj > {{path/to/output_file}}`
 
-- Using more efficient algorithm to intersect two pre-sorted files:
+- Using more efficient algorithm to intersect two pre-[sorted] files:
 
 `bedtools intersect -a {{path/to/file1}} -b {{path/to/file2}} -sorted > {{path/to/output_file}}`
 
-- Group file `path/to/file` based on the first three and the fifth column and summarize the sixth column by summing it up:
+- [g]roup file `path/to/file` based on the first three and the fifth [c]olumn and apply the sum [o]peration on the sixth column:
```
Comment (acuteenvy) [inline pages/common/bedtools.md:19]:
```suggestion
- [g]roup a file based on the first three and the fifth [c]olumn and apply the sum [o]peration on the sixth column:
```

File: pages/common/boxes.md [hunk 3]
```diff
@@ -7,18 +7,26 @@
...
+- [a]lign the box text [h]orizonally (at [l]eft, [c]enter or [r]ight):
 
-`echo "{{string}}" | boxes -a c`
+`echo "{{string}}" | boxes -a h{{l|c|r}}`
+
+- [a]lign the box text [v]ertically (at [t]op, [c]enter or [b]ottom):
+
+`echo "{{string}}" | boxes -a v{{t|c|b}}`
+
+- [j]ustificate the box text (at [l]eft, [c]enter or [r]ight):
```
Comment (acuteenvy) [inline pages/common/boxes.md:30]:
```suggestion
- [j]ustify the box text (at [l]eft, [c]enter or [r]ight):
```

File: pages/common/bup.md [hunk 4]
```diff
@@ -3,22 +3,22 @@
...
+- Backup a [d]irectory to the repository specifying its [n]ame:
 
 `bup -d {{path/to/repository}} save -n {{backup_name}} {{path/to/directory}}`
 
 - Show the backup snapshots currently stored in the repository:
 
 `bup -d {{path/to/repository}} ls`
 
-- Restore a specific backup snapshot to a target directory:
+- Restore a specific backup snapshot to a target directory ([-C]):
```
Comment (acuteenvy) [inline pages/common/bup.md:22]:
```suggestion
- Restore a specific backup snapshot to a target dire[C]tory:
```

File: pages/common/bzip3.md [hunk 5]
```diff
@@ -7,11 +7,11 @@
 
 `bzip3 {{path/to/file_to_compress}}`
 
-- Decompress a file:
+- [d]ecompress a file:
 
 `bzip3 -d {{path/to/compressed_file.bz3}}`
 
-- Decompress a file to `stdout`:
+- Decompress a file to `stdout` ([c]):
```
Comment (acuteenvy) [inline pages/common/bzip3.md:14]: Is this kind of mnemonic usage documented? I can't find anything in the style guide.
```suggestion
- Decompress a file to `stdout`:
```

File: pages/common/cancel.md [hunk 6]
```diff
@@ -20,14 +20,14 @@
...
 
 `cancel -a`
 
-- Cancel all jobs of a specific printer:
+- Cancel [a]ll jobs of a specific printer:
 
 `cancel -a {{printer}}`
 
-- Cancel the current job of a specific server and then delete job data files:
+- Cancel the current job of a specific server and then e[x]clude job data files:
```
Comment (acuteenvy) [inline pages/common/cancel.md:31]: Delete and exclude both mean different things.

https://openprinting.github.io/cups/doc/man-cancel.html
> -x
> Deletes job data files in addition to canceling. 
```suggestion
- Cancel the current job of a specific server and then delete job data files:
```

File: pages/common/certutil.md [hunk 7]
```diff
@@ -3,22 +3,22 @@
...
+- Create a [N]ew certificate database in the current [d]irectory:
 
 `certutil -N -d .`
 
 - List all certificates in a database:
 
 `certutil -L -d .`
 
-- List all private keys in a database:
+- List all private [K]eys in a database specifying the passwords [f]ile:
```
Comment (acuteenvy) [inline pages/common/certutil.md:14]:
```suggestion
- List all private [K]eys in a database specifying the password [f]ile:
```

File: pages/common/certutil.md [hunk 8]
```diff
@@ -3,22 +3,22 @@
...
 
 `certutil -K -d . -f {{path/to/password_file.txt}}`
 
-- Import the signed certificate into the requesters database:
+- [A]dd the signed certificate to the requesters database specifying a [n]ickname, [t]rust attributes and an [i]nput CRT file:
 
 `certutil -A -n "{{server_certificate}}" -t ",," -i {{path/to/file.crt}} -d .`
 
-- Add subject alternative names to a given certificate:
+- Add subject alternative names to a given [c]ertificate with a specific key size ([g]):
```
Comment (acuteenvy) [inline pages/common/certutil.md:22]:
```suggestion
- Add subject alternative names to a given [c]ertificate with a specific key size:
```

File: pages/common/certutil.md [hunk 9]
```diff
@@ -3,22 +3,22 @@
...
 
-`certutil -S -f {{path/to/password_file.txt}} -d . -t ",," -c "{{server_certificate}}" -n "{{server_name}}" -g {{2048}} -s "CN={{common_name}},O={{organization}}"`
+`certutil -S -f {{path/to/password_file.txt}} -d . -t ",," -c "{{server_certificate}}" -n "{{server_name}}" -g {{512..16384}} -s "CN={{common_name}},O={{organization}}"`
```
Comment (acuteenvy) [inline pages/common/certutil.md:24]: I would suggest leaving 2048 in there. The default key type is RSA, and 512-bit RSA is ***very insecure***.
```suggestion
`certutil -S -f {{path/to/password_file.txt}} -d . -t ",," -c "{{server_certificate}}" -n "{{server_name}}" -g {{2048}} -s "CN={{common_name}},O={{organization}}"`
```

File: pages/common/banner.md [hunk 10]
```diff
@@ -7,9 +7,9 @@
 
 `banner "{{Hello World}}"`
 
-- Print the text message as a banner with a width of 50 characters:
+- Use a 50 characters banner [w]idth:
```
Comment (kbdharun) [inline pages/common/banner.md:10]: Maybe we can rephrase this?

```suggestion
- Use a banner [w]idth of 50 characters:
```

File: pages/common/bzip3.md (hunk 5 above)
Comment (vitorhcl) [reply pages/common/bzip3.md:14]: Nope, I've seen this in some pages, but we can document this.

Comment (vitorhcl) [reply pages/common/bzip3.md:14]: Remember that the purpose of the guides are to **describe** our practices, not to **prescribe** them.

The style guide doesn't say anything about this. In my opinion, mnemonics between parenthesis are nice when there are multiple options in the example command.

Comment (kbdharun) [reply pages/common/bzip3.md:14]: > Is this kind of mnemonic usage documented?

It isn't documented but we use this in a few translations of popular commands where we need to document the short option or context.

**Review by kbdharun — APPROVED** [review]

File: pages/common/bzip3.md (hunk 5 above)
Comment (kbdharun) [reply pages/common/bzip3.md:14]: I will try to create a PR to document this in the style guide later today.

**Pull request merged**